.circleci/config.yml

```diff
@@ -33,6 +33,9 @@ jobs:
             echo "@qlik-trial:registry=https://npm.pkg.github.com/
             //npm.pkg.github.com/:_authToken=$GITHUB_TOKEN
             //npm.pkg.github.com/:always-auth=true" >> ~/.npmrc
+      - run:
```

Comment:
shouldn't you do this after you install? Or am I misunderstanding how deduplicate works?

Reply:
It does not matter.  As `npx yarn-deduplicate` check yarn.lock file and `yarn install --frozen-lockfile` does not change yarn.lock file